cookbook-react/recipes/tab-bars/tab-bar-1/styles.scss

```diff
@@ -0,0 +1,68 @@
+$items: 3;
```

Comment:
If you do the change I mentioned in the file before, I would do something like this:
```
$items: var(--number-of-items, 3);
```
And you can send `options.length` to `--number-of-items`.